test/file.ts

```diff
@@ -3654,22 +3654,6 @@ describe('File', () => {
       });
     });
 
-    it('should execute callback with error & API response', done => {
```

Comment:
For future reference, spoke with @ddelgrosso1 and determined that this case does not exist and that it would be safe to remove this test.